WcaOnRails/app/views/results/_index_by_region_table.html.erb

```diff
@@ -0,0 +1,60 @@
+<%= wca_table table_class: "wca-results" do %>
+  <thead>
+    <tr>
+      <% columns_number = 5 %>
+      <th class="region"> <%= t("results.index.table_elements.region") %> </th>
+      <th class="result"> <%= t("results.index.table_elements.result") %> </th>
+      <th class="name"> <%= t("results.index.table_elements.name") %> </th>
+      <th class="competition"> <%= t("results.index.table_elements.competition") %> </th>
+      <% if @is_average %>
+        <% columns_number += 5 %>
+        <th class="solves" colspan="5"> <%= t("results.index.table_elements.solves") %> </th>
+      <% end %>
+
+      <!-- Extra column for .table-greedy-last-column -->
+      <th></th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% shown = {} %>
+    <% @rows_to_display.each_with_index do |row, i| %>
+      <% result = LightResult.new(row, row["countryId"], row["formatId"], row["roundTypeId"], params[:event_id]) %>
```

Comment:
All the other arguments are included in `row`, so my idea was to just pass the row ^^

Reply:
Right, that makes sense :slightly_smiling_face: I've changed it already, adding `result.eventId       eventId,` to one of the queries that was missing it!